docs/bot/setup_debian.md

```diff
@@ -0,0 +1,136 @@
+# Setting up Draupnir on Debian
+These are instructions for the installation of draupnir from source on Debian  
+This installation method is intended for experienced sysadmins.
+
+## installation
+install git curl and sudo
+```shell
+apt update && apt install -y git curl sudo
+```
+install node 18 from the node source repo, the full instructions can be found at https://github.com/nodesource/distributions
+```shell
+curl -fsSL https://deb.nodesource.com/setup_18.x -o nodesource_setup.sh
+bash nodesource_setup.sh
+apt update && apt install nodejs -y
+```
+install yarn via npm
+```shell
+npm install --global yarn                                                                                                                    # install yarn via npn
+```
+create the directory to clone the repo
+```shell
+mkdir /opt/mod-bot  
+```
+clone the repo
+```shell
+git clone https://github.com/the-draupnir-project/Draupnir.git /opt/mod-bot/Draupnir
+```
+create the directory for draupnirs datastorage
+```shell
+mkdir /opt/mod-bot/Draupnir/datastorage
+```
+add typescript and corepack
+```shell
+yarn global add typescript
```

Comment:
is adding typescript necessary? did draupnir fail to run when you did that? Could indicate that we're doing something wrong on our end if so.

Reply:
you are right, i just built it without adding typescript, not sure why i had it in there.....

Reply:
i edited it

Reply:
thankies